src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -15,6 +16,41 @@
 from promptflow.core._utils import render_jinja_template_content
 
 
+@dataclass
+class PromptyModelConfiguration:
+    api: str
+    configuration: dict
+    parameters: dict
+    response: str = "first"
+    connection: InitVar[Union[str, _Connection]] = None
```

Comment:
I think the connection will be in configuration dict following @D-W- 's design

Reply:
move connection to configuration dict